libraries/AP_HAL_Linux/Heat_Pwm.cpp

```diff
@@ -74,4 +78,9 @@ void HeatPwm::set_imu_temp(float current)
     _last_temp_update = AP_HAL::millis();
 }
 
+void HeatPwm::set_imu_target_temp(int8_t *target)
```

Comment:
Is this a pointer so that you can turn it off? In that case, souldn't we actually turn it off by disabling the pwm? Otherwise I don't see a reason for this being a pointer